src/base/BaseScene.cc

```diff
@@ -118,17 +118,38 @@ std::string BaseScene::Name() const
 }
 
 //////////////////////////////////////////////////
-common::Time BaseScene::SimTime() const
+std::chrono::steady_clock::duration BaseScene::Time() const
 {
   return this->simTime;
 }
 
 //////////////////////////////////////////////////
-void BaseScene::SetSimTime(const common::Time &_time)
+void BaseScene::SetTime(const std::chrono::steady_clock::duration &_time)
 {
   this->simTime = _time;
 }
 
+//////////////////////////////////////////////////
+common::Time BaseScene::SimTime() const
+{
+  std::pair<int64_t, int64_t> secsAndNsecs =
+    math::durationToSecNsec(this->simTime);
+#ifndef _WIN32
+# pragma GCC diagnostic push
+# pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
```

Comment:
Technically these aren't necessary because `ign-rendering4` is using `ign-common3`, which still supports `common::Time`.